app/models/prison.rb

```diff
@@ -1,9 +1,34 @@
 class Prison < ApplicationRecord
-  validates :prison_type, presence: true
-  validates :code, :name, presence: true, uniqueness: true
+  validates :prison_type, :code, presence: true
```

Comment:
this looks like a typo? This line should be unaltered - code is already the primary key